server/controllers/cohort.js

```diff
@@ -248,13 +250,98 @@ const removeFromFavourites = (req, resp) => {
   );
 };
 
+const getMembers = (req, resp) => {
+  const { id: cohortId } = req.params;
+
+  Cohort.findOne({ _id: cohortId }, 'memberIds ownerIds', (err, doc) => {
+    if (err) {
+      return resp.status(400).send({
+        message: "Can't get members data. Please try again."
+      });
+    }
+
+    if (doc) {
+      const { memberIds, ownerIds } = doc;
+
+      User.find({ _id: [...memberIds, ownerIds] }, (err, docs) => {
+        if (err) {
+          return resp.status(400).send({
+            message: "Can't get members data. Please try again."
+          });
+        }
+        const users = responseWithUsers(docs, ownerIds);
+        resp.status(200).json(users);
+      });
+    }
+  });
+};
+
+const addMember = (req, resp) => {
+  const {
+    user: { _id: currentUser }
+  } = req;
+  const { id: cohortId } = req.params;
+  const { email } = req.body;
+
+  Cohort.findOne({ _id: cohortId, $or: [{ ownerIds: currentUser }] })
+    .exec()
+    .then(doc => {
+      if (doc) {
+        return User.findOne({ email })
+          .exec()
+          .then(doc => {
+            const { _id, avatarUrl, displayName, email } = doc;
+            return { _id, avatarUrl, displayName, email };
+          })
+          .catch(() =>
+            resp.status(400).send({ message: 'User data not found.' })
+          );
+      }
+      return resp
+        .status(400)
+        .send({ message: "You don't have permission to add new member" });
+    })
+    .then(data => {
+      const { _id: newMemberId, displayName, avatarUrl, email } = data;
+      Cohort.findOneAndUpdate(
+        { _id: cohortId, memberIds: { $nin: [newMemberId] } },
+        { $push: { memberIds: newMemberId } }
+      )
+        .exec()
+        .then(doc => {
+          if (doc) {
+            return resp
+              .status(200)
+              .json({ avatarUrl, displayName, email, newMemberId });
```

Comment:
isOwner: false should be sent here, because it's needed and in other cases is sent e.g getMembers controller. Create some helper method responseWithUser like in other cases